contributors/guide/owners.md

```diff
@@ -40,6 +40,10 @@ OWNERS files are in YAML format and support the following keys:
     Allows the use case where `a/deep/nested/OWNERS` file prevents `a/OWNERS` file from having any
     effect on `a/deep/nested/bit/of/code`
 - `reviewers`: a list of GitHub usernames or aliases that are good candidates to `/lgtm` a PR
+- `security_contacts`: a list of users to contact for security issues
+  - `github`: GitHub username (required)
+  - `email`: E-mail address (optional)
+  - NB: the contacts here are for use _by_ the **[Kubernetes PSC][PSC]**, and should not normally include PSC members. The PSC will use the contact information to coordinate security issues.
```

Comment:
One last question, but is NB in this context?

Overall though lgtm

Reply:
Contextually, this is an abbreviation for the latin "Nota Bene", i.e. "mark my words".

Probably it should just say "Note:" to avoid future confusion?

Reply:
Yeah, I would definitely say note here,.